packages/search-ui/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "target": "es2015",
+    "lib": [
+      "dom",
+      "dom.iterable",
```

Comment:
I pasted this from an existing confguration of mine but im going to omit lib configuration because not sure its needed anymore - https://www.typescriptlang.org/tsconfig#lib. typescript has some smart defaults for these values.